nav2_bringup/bringup/launch/tb3_simulation_launch.py

```diff
@@ -129,9 +129,9 @@ def generate_launch_description():
         'world',
         # TODO(orduno) Switch back once ROS argument passing has been fixed upstream
         #              https://github.com/ROBOTIS-GIT/turtlebot3_simulations/issues/91
-        # default_value=os.path.join(get_package_share_directory('turtlebot3_gazebo'),
-        #                            'worlds/turtlebot3_worlds/waffle.model'),
-        default_value=os.path.join(bringup_dir, 'worlds', 'waffle.model'),
+        default_value=os.path.join(get_package_share_directory('turtlebot3_gazebo'),
```

Comment:
If this TODO isn't needed any more, we should change this in another PR. Lets not change unrelated things in this one.